index.html

```diff
@@ -461,5 +461,23 @@ <h2 class="fl-mapviz-hospital-title">
             searchResultTemplateUrl: "src/html/searchResultTemplate.html"
         });
     </script>
+
```

Comment:
Why did all these extra blank lines appear?

Reply:
Blank lines are removed. It was because every run of `node buildSVG.js` adds a couple of new blank lines after the `</style>` tag.

I debugged `buildSVG.js`. At line 74 when JSDOM converts the file content to DOM structure, these new lines are added. Unfortunately, I didn't find a solution for it.